Testreglar/3.3.3/App/app-3.3.3a.json

```diff
@@ -0,0 +1,233 @@
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har appsiden skjema/skjemaelement?",
+			"ht": "<p><strong>Merk:</strong> Du skal ikke teste skjema i PDF, Word eller lignende.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Appsiden har ikke skjema/skjemaelement."
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Er det mulig å sveipe til skjemaelementer?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Aktiver skjermleser (opplesningsfunksjon)\n<ul>\n<li>iOS: VoiceOver</li>\n<li>Android: Talkback</li>\n</ul>\n</li>\n<li>Åpne appsiden du tester</li>\n<li>Sjekk om det er mulig å sveipe til minst et skjemaelement</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ikkje testbart",
+					"utfall": "Det er ikke mulig å sveipe til skjemaelementer på appsiden."
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.5"
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hvilket skjema/skjemaelement tester du?",
+			"ht": "<p>Beskriv skjema/skjemaelement, slik at det er mulig å identifisere det i ettertid. </p>\n<p><strong>Merk:</strong> Hvis skjema/skjemaelement inngår i en prosess skal du også beskrive prosessen.</p>",
+			"type": "tekst",
+			"label": "Skjema/prosess:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.5",
+			"spm": "Oppdager skjemaet automatisk at utfylt inndata har feil format eller verdi?",
+			"ht": "<p>Slik tester du:</p>\n<ul>\n<li>Legg inn feil inndata i alle skjemaelement.\n<ul>\n<li>Hvis skjemaelementer er forhåndsutfylt, endre forhåndsutfylt informasjon til feil inndata format eller verdi der det er mulig.</li>\n<li>Forhåndsutfylt informasjon betyr at det allerede står inndata i skjemaet når appsiden er lastet inn.</li>\n<li>I sitasjon appsiden krever informasjon, men som brukeren har utelatt, for eksempel du ikke har krysset av i en avkryssingsboks</li>\n</ul>\n</li>\n<li>Prøv deretter å fullføre/sende inn eller gå videre i skjemaet.</li>\n</ul>\n<p><strong>Merk:</strong> Skjema der knappen for å gå videre eller fullføre er deaktivert, skal testes.</p>\n<p>Et deaktivert skjemaelement for å gå videre eller fullføre skjemaet, som blir aktivt når brukeren fyller ut informasjon rett format eller verdi, indikerer at brukeren har fylt ut inndata som faller utenfor påkrevd dataformat eller verdier, og at dette oppdages automatisk, uten at brukeren er involvert.</p>\n<p>Eksempel på feil inndata er:</p>\n<ul>\n<li>Postnummer som inneholder bokstaver.</li>\n<li>Navn som inneholder tall eller spesialtegn.</li>\n<li>Ugyldig e-postformat.</li>\n<li>Ugyldig datoformat.</li>\n<li>Ugyldig telefonnummer.</li>\n<li>Data utenfor grenseverdi, for eksempel måned 14 i en dato. </li>\n</ul>",
```

Comment:
Samme kommentar som i nett, savner kilde til "alle element som inngår i formatet, stå i forslaget"

Reply:
Dette er vår tolkning basert på forstå artikkelen og G177. "suggestions for the correct text" må vise hele format og ikke bare en bokstav eller tegn for eksempel @ eller telefon tegn.